cmake/Modules/MacroConfigureMSVCRuntime.cmake

```diff
@@ -1,36 +0,0 @@
-macro(configure_msvc_runtime)
```

Comment:
Isn't a good candidate for extracting?
You can extract all similar (more or less) trivial cleanups into separate PR.

Reply:
@glassez same reasoning as  https://github.com/qbittorrent/qBittorrent/pull/13327#discussion_r485157109.